apps/host/src/index.css

```
@@ -0,0 +1,24 @@
+
+:root {
+    --outer-space: 1.5rem;
+}
+
+body {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 400;
+}
+
+html, body, app {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)